packages/dds/merge-tree/src/test/beastTest.ts

```diff
@@ -820,11 +813,11 @@ export function TestPack(verbose = true) {
         let errorCount = 0;
 
         // function asyncRoundStep(asyncInfo: AsyncRoundInfo, roundCount: number) {
-        //     if (asyncInfo.state == AsyncRoundState.Insert) {
+        //     if (asyncInfo.state === AsyncRoundState.Insert) {
```

Comment:
Do you know why this code is commented?

Reply:
nope. just some old test code. the test still runs, and has found bugs, but not actively